kroxylicious-filters/kroxylicious-multitenant/src/main/java/io/kroxylicious/proxy/filter/multitenant/MultiTenantTransformationFilter.java

```diff
@@ -487,16 +486,18 @@ private String removeTenantPrefix(FilterContext context, String brokerSideName)
         return brokerSideName.substring(tenantPrefix.length());
     }
 
-    private static String getTenantPrefix(FilterContext context) {
+    private String getTenantPrefix(FilterContext context) {
```

Comment:
we should probably fail early if the virtualClusterName contains characters that aren't valid for a kafka resource name.